### PR TITLE
Feature update delivery address 2

### PR DESCRIPTION
**Pull request by infamouskeyduster**
# PR Description
Please include a summary of the change:
- This PR is our attempt at making a FE => BE request for shipments data
- We could not resolve a 400 error, and could not retrieve any shipments data for our user

Commit Emojis:
:crystal_ball: JS modifications/updates.
:beetle: If it is a bug or issue fix, describe the issue that was fixed.

Note any questions or struggles as well.
- Massive struggles putting our FE request in to BE; could not get user's shipment data returned.

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the Turing style guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: code/web/src/modules/user/EditProfile.js [hunk 1]
```diff
@@ -223,18 +219,17 @@ class EditProfile extends Component {
 }
 
 // Component Properties
-// EditProfile.propTypes = {
-//   user: PropTypes.object.isRequired,
-//   logout: PropTypes.func.isRequired
-// }
+EditProfile.propTypes = {
```
Comment (rachael-t) [inline code/web/src/modules/user/EditProfile.js:222]: Great job getting this added back!

File: code/web/src/modules/user/Profile.test.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ describe("Profile", () => {
         const userName = getByText("Celine Dion");
         const userDescription = getByText("Recently stepped up my fashion game and love showing off dramatic outfits to match my music");
         const userEmail = getByText("heartgoeson@crate.com");
-        const userAddress = getByText("123 Crate St. Boulder, CO 80302", {exact: false})
```
Comment (rachael-t) [inline code/web/src/modules/user/Profile.test.js:47]: Thanks for fixing this!

File: code/web/src/modules/user/api/actions.js [hunk 3]
```diff
@@ -145,20 +145,26 @@ export function updateUserProfile(user) {
   }
 }
 
-// Get Shipment Date Data 
-export function getShipmentData(user) {
+// Get Shipment Date
```
Comment (rachael-t) [inline code/web/src/modules/user/api/actions.js:148]: Solid attempt at this!

**Review by atkinson-daniel — APPROVED** [review]

**Review by rachael-t — APPROVED** [review]
Great job trying to get this in the last few hours!

**Pull request merged**